pkg/pillar/vendor/github.com/packetcap/go-pcap/pcap_linux.go

```diff
@@ -287,6 +301,8 @@ func openLive(iface string, snaplen int32, promiscuous bool, timeout time.Durati
 	})
 	logger.Debug("started")
 	h := Handle{
+		// we start with it not open
+		isOpen:   1,
```

Comment:
Wait, if that's the semantics it should be called closed and not isOpen.

Reply:
It is an internal-to-pcap thing, so happy to review over there, but this PR has it functional. Let's let it go for now.

Reply:
If you think it should be called isOpen, then why not reverse the logic and initialize to zero and set to 1 when opened and set to zero when closed.
Otherwise any other reviewer will be confused.

Reply:
Honestly? Because updating vendored module versions in pillar is a bit of a pain, and I didn't want to do it again.

1. `make shell`
1. manually find the lines that ref the old version in `go.mod`/`go.sum` and remove them
1. run `go mod vendor`
1. remove all of the other things that got update on the way, usually `eve/api/go` but sometimes other things
1. Check that there were no side effect changes in `go.mod`/`go.sum` and in `vendor/modules.txt`

A lot harder than just `go mod update mypackage@master` or similar.

I did it anyways. Check it now.